doc/ref/pperm.xml

```diff
@@ -1551,11 +1551,11 @@ multiplicative neutral element (i.e. an identity element) but not to satisfy
 <Example><![CDATA[
 gap> f := PartialPerm( [ 1, 2, 3, 6, 8, 10 ], [ 2, 6, 7, 9, 1, 5 ] );;
 gap> S := Semigroup(f, One(f));
-<commutative partial perm monoid of rank 9 with 1 generator>
+<partial perm semigroup of rank 9 with 2 generators>
 gap> IsMonoid(S);
-true
+false
 gap> IsPartialPermMonoid(S);
-true]]></Example>
+false]]></Example>
```

Comment:
The paragraph preceding this example show that there has been a problem for a long time:

> ```Note that it is possible for a partial perm semigroup to have a multiplicative neutral element (i.e. an identity element) but not to satisfy <C>IsPartialPermMonoid</C>. For example,```

So it's good to get this sorted.

I like the general change of `Semigroup` not producing `IsMonoid` objects any more. I'll be happy for this PR to be merged.

I've long been aware that semigroups of partial permutation are weird with respect to `One`: the `One` of a partial perm does not necessarily match the `One` of any inverse monoid to which it belongs. But I've never found a good way of resolving this problem.

Reply:
I can no longer make sense of my comment here (at least the first half) so I've hidden it.